pkg/ccl/changefeedccl/show_changefeed_jobs_test.go

```diff
@@ -116,6 +119,43 @@ func TestShowChangefeedJobsBasic(t *testing.T) {
 	cdcTest(t, testFn, feedTestOmitSinks("webhook", "sinkless"), feedTestNoExternalConnection)
 }
 
+func TestShowChangefeedJobsShowsHighWaterTimestamp(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	testFn := func(t *testing.T, s TestServer, f cdctest.TestFeedFactory) {
+		sqlDB := sqlutils.MakeSQLRunner(s.DB)
+		valRange := []int{1, 1000}
+		sqlDB.Exec(t, `CREATE TABLE foo(a INT PRIMARY KEY)`)
+		foo := feed(t, f, `CREATE CHANGEFEED FOR foo WITH resolved='0s',min_checkpoint_frequency='0s'`)
+		sqlDB.Exec(t, fmt.Sprintf(`INSERT INTO foo (a) SELECT * FROM generate_series(%d, %d)`, valRange[0], valRange[1]))
+
+		defer func() {
+			closeFeed(t, foo)
+		}()
+
+		var highWaterTimestamp gosql.NullFloat64
+		var readableHighWaterTimestamp gosql.NullTime
+
+		// Wait for the high water timestamp to be non-null.
+		testutils.SucceedsSoon(t, func() error {
+			sqlDB.QueryRow(t, "SELECT high_water_timestamp, readable_high_water_timestamp from [SHOW CHANGEFEED JOBS]").Scan(&highWaterTimestamp, &readableHighWaterTimestamp)
+
+			if !highWaterTimestamp.Valid {
+				return errors.Errorf("high water timestamp not populated: %v", highWaterTimestamp)
+			}
+
+			return nil
+		})
+
+		// Timestamps in CockroachDB have microsecond precision by default
+		highWaterTimestampTime := time.Unix(0, int64(highWaterTimestamp.Float64)).UTC().Round(time.Microsecond)
+		require.Equal(t, highWaterTimestampTime, readableHighWaterTimestamp.Time)
+	}
+
+	cdcTest(t, testFn, feedTestOmitSinks("sinkless"))
```

Comment:
Are there any specific reasons that we are omitting sinkless? Is it because sinkless does not use the job system table?

Reply:
It seems to have worked without omitting sinkless so I removed that third argument